### PR TITLE
Add ZeroTier "Get Status" action; update ZT documentation

### PR DESCRIPTION
**Pull request by zalmanlew**
Other fixes include typos and corrections in the documentation

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

**Comment by dannyroosevelt** [comment]
Thank you @zalmanlew! Is there a reason to remove links to ZeroTier's docs for each of those actions?

**Comment by zalmanlew** [comment]
@dannyroosevelt they don't format correctly (as markdown) and aren't clickable; the "info" they provide I was able to copy from the summary in the docs. 
All of the actions right now are GET actions, so it just responds with an object. Once we have actions which can make changes, it _would_ be useful to include API doc links.

I removed them from the triggers as they aren't informative in those cases.

**Comment by zalmanlew** [comment]
@dannyroosevelt here's how they currently format
image

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

File: components/zerotier/actions/get-network/get-network.mjs [hunk 1]
```diff
@@ -3,9 +3,9 @@ import zerotier from "../../zerotier.app.mjs";
 export default {
   key: "zerotier-get-network",
   type: "action",
-  version: "0.0.1",
+  version: "0.0.2",
   name: "Get Network",
-  description: "Get a specific network. [See docs here](https://docs.zerotier.com/central/v1/#operation/getNetworkByID)",
+  description: "Get a specific network. Returns a single network.",
```
Comment (jcortes) [inline components/zerotier/actions/get-network/get-network.mjs:8]: Can you plese bring back again all the doc references in the description of each component? Thanks!
```suggestion
  description: "Get a specific network. Returns a single network. [See docs here](https://docs.zerotier.com/central/v1/#operation/getNetworkByID)",
```

**Review by jcortes — CHANGES_REQUESTED** [review]
Hi @zalmanlew I've just addeded one minor request change the rest looks good to me!

File: components/zerotier/actions/get-network/get-network.mjs (hunk 1 above)
Comment (zalmanlew) [reply components/zerotier/actions/get-network/get-network.mjs]: @jcortes done

**Review by dannyroosevelt — APPROVED** [review]

**Pull request merged**